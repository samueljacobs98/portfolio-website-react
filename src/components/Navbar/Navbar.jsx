import "./Navbar.scss";
const Navbar = () => {
  return (
    <ul className="navbar">
      <li>
        <a className="navbar__link" href="#projects">
          Projects
        </a>
      </li>
      <li>
        <a className="navbar__link" href="#about">
          About
        </a>
      </li>
      <li>
        <a className="navbar__link" href="#contact">
          Contact
        </a>
      </li>
    </ul>
  );
};

export default Navbar;
