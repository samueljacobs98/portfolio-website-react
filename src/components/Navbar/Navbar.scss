@use "../../assets/scss/variables" as *;
.navbar {
  display: flex;
  align-items: center;

  list-style-type: none;

  &__link,
  &__link:visited {
    text-decoration: none;
    color: $color-white;
    margin-left: $margin-nav;
    font-size: $font-size;
  }
  &__link:hover {
    text-decoration: underline;
    font-weight: bold;
  }
}
