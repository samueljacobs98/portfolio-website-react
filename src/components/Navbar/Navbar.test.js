import { render } from "@testing-library/react";
import Navbar from "./Navbar";

test("should match snapshot", () => {
  const { container } = render(<Navbar />);
  expect(container).toMatchSnapshot();
});
