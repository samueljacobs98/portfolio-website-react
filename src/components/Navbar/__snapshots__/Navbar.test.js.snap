// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <ul
    class="navbar"
  >
    <li>
      <a
        class="navbar__link"
        href="#projects"
      >
        Projects
      </a>
    </li>
    <li>
      <a
        class="navbar__link"
        href="#about"
      >
        About
      </a>
    </li>
    <li>
      <a
        class="navbar__link"
        href="#contact"
      >
        Contact
      </a>
    </li>
  </ul>
</div>
`;
